wiki/Play_Styles/en.md

```diff
@@ -162,8 +162,17 @@ The play-style tuned for seasoned veterans at conveyor-type rhythm games.
 
 Since 8 Keys requires mapper's decision or Game Modifiers, most players owning these arcade controllers can handle up to 7Keys (max default) and 1 special.
 
+### Dancepad
+
+Dancepads are controllers for games like the [Dance Dance Revolution](https://wikipedia.org/wiki/Dance_Dance_Revolution) series and the [Pump it Up](https://en.wikipedia.org/wiki/Pump_It_Up_(video_game_series)) series. ([Stepmania](https://wikipedia.org/wiki/StepMania) is a semi-well known community driven clone of both of these games that people like to use dance pads on). 
+
+Players dance on the dancepad to input commands. DDR pads have 4 panels: Up, Down Left and Right. DDR Solo pads add and Up-left and Up-right to that. PIU pads have 5 panels, Up-left, Up-right, Down-left, Down-Right and Center. 9 panel pads also exist, for compatibility with both PIU and DDR without having to change a pad. Normal DDR and PIU pads are often used in doubles mode in their respective games, where one player uses 2 pads for a song. DDR Solo games were all single player, so they aren't commonly doubled. 
```

Comment:
Uppercase all the directions (i.e. `Up-left` -> `Up-Left`).

`Up, Down Left and Right` -> `Up, Down, Left, and Right`
Add comma before `and` on lists. Check other instances.

`DDR Solo pads add and` -> `DDR Solo pads add`